openedx/core/djangoapps/kafka_consumer/management/commands/consume_events.py

```diff
@@ -0,0 +1,94 @@
+"""
+Management command for listening to license-manager events and logging them
+"""
+
+import logging
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from confluent_kafka import KafkaError, DeserializingConsumer
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.serialization import StringDeserializer
+from openedx.core.djangoapps.kafka_consumer.consumers import getHandler
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Listen for license events from the event bus and log them. Only run on servers where KAFKA_ENABLED is true
+    """
+    help = """
+    This starts a Kafka event consumer that listens to the specified topic and logs all messages it receives. Topic
+    is required.
+
+    example:
+        manage.py ... consume_events -t license-event-prod
+
+    """
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            '-t', '--topic',
+            nargs=1,
+            required=True,
+            help='Topic to consume'
+        )
+
+        parser.add_argument(
+            '-g', '--group_id',
+            nargs=1,
+            required=True,
+            help='Consumer group id'
+        )
+
+    def handle(self, *args, **options):
+        if not settings.KAFKA_ENABLED:
+            logger.error("Kafka not enabled")
+            return
+        try:
+            KAFKA_SCHEMA_REGISTRY_CONFIG = {
+                'url': settings.SCHEMA_REGISTRY_URL,
+                'basic.auth.user.info': f"{settings.SCHEMA_REGISTRY_API_KEY}"
+                f":{settings.SCHEMA_REGISTRY_API_SECRET}",
```

Comment:
You can tell my brain was having trouble parsing. Could you do something like the following?
```suggestion
                'basic.auth.user.info':
                    f"{settings.SCHEMA_REGISTRY_API_KEY}:{settings.SCHEMA_REGISTRY_API_SECRET}",
```
If it can't fit in one string for some reason:
1. Could you move the colon to the end of the first string?
2. Could you add a tab before the second string?